base/.eslintrc.json

```diff
@@ -56,6 +66,17 @@
     "default-case": 0,
     "no-plusplus": 0,
     "dot-notation": 0,
-    "eol-last": 2
+    "eol-last": 2,
```

Comment:
I ran this tool on this config https://github.com/prettier/eslint-config-prettier#cli-helper-tool
and the output was:
```
- @typescript-eslint/member-delimiter-style
- @typescript-eslint/type-annotation-spacing
- comma-dangle
- comma-spacing
- comma-style
- dot-location
- eol-last
- indent
- keyword-spacing
- linebreak-style
- no-extra-semi
- no-mixed-spaces-and-tabs
- no-multi-spaces
- no-multiple-empty-lines
- no-trailing-spaces
- react/jsx-closing-bracket-location
- react/jsx-curly-spacing
- react/jsx-equals-spacing
- react/jsx-first-prop-new-line
- react/jsx-indent-props
- react/jsx-max-props-per-line
- react/jsx-tag-spacing
- semi
- semi-spacing
- space-before-blocks
- space-infix-ops
- space-unary-ops
```
So don't think we can use `eol-last` or `comma-dangle`? Not sure though

Reply:
`comma-dangle` is the eslint version of `trailing-comma` which was translated from `tslint-config-dabapps` and `eol-last` was there already.

Let's have a discussion about what rules to include/exclude in the FE strategy meeting.

Reply:
sure